torch/_torch_docs.py

```diff
@@ -2580,6 +2580,7 @@ def merge_dicts(*dicts):
 add_docstr(torch.isinf,
            r"""
 Returns a new tensor with boolean elements representing if each element is `+/-INF` or not.
+Complex values are infinite when their real or imaginary part is infinite.
```

Comment:
nit -  `real and(or) imaginary part is infinite`

Reply:
Sure.